src/main/java/commands/SummaryCommand.java

```diff
@@ -0,0 +1,98 @@
+package commands;
+
+import records.*;
+import java.util.*;
+import manager.ExpenseManager;
+
+
+public class SummaryCommand extends Command {
+    /** Keyword to trigger summary command. */
+    public static final String COMMAND_WORD = "summary";
+
+    /** Help message for summary command. */
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Shows the summary table based on their type."
+            + "\nExample: " + COMMAND_WORD;
+
+    /**
+     * Executes the command and returns the result.
+     *
+     * @return Result of the command
+     */
+    @Override
+    public CommandResult execute() {
+        List<Record> allRecords = recordMgr.getAllRecords();
+        List<String> types=Arrays.asList("fashion", "food", "accessory","others");
+        for (String type:types) {
+            int flag=0;
```

Comment:
Please try to keep the space before and after the operators consistent in the scripts.